model/facade/action/GetPersonalSummaryByLoginDateAction.php

```diff
@@ -103,6 +103,17 @@ private function getWorkableHoursInThisJourneyPeriod() {
     }
 
     /**
+     * @return float
+     */
+    private function getNumberOfWeeksInThisJourney() {
+        $initDate = $this->currentJourney->getInitDate();
+        $endDate = $this->currentJourney->getEndDate();
+
+        $interval = $initDate->diff( $endDate );
+        return round($interval->days/7, 1);
```

Comment:
You are getting a float with one decimal digit, and I think you should get an integer. Shouldn't this be `ceil()`?